package.json

```
@@ -2,7 +2,7 @@
   "name": "shebang-language-associator",
   "displayName": "Shebang Language Associator",
   "description": "A tiny extension to associate shebang lines with languages",
-  "version": "1.2.0",
+  "version": "1.2.1",
```

Nit: I think really this is a new feature so let's bump the minor version.

```suggestion
  "version": "1.3.0",
```